src/modules/Builder/features/SaveAndPublish/SaveAndPublish.utils.ts

```diff
@@ -46,6 +50,7 @@ export const removeAppletExtraFields = () => ({
   id: undefined,
   theme: undefined,
   version: undefined,
+  undefined,
```

Comment:
?

Reply:
to deal with { undefined: { items: [] } } problem we fixed before due to desynchronization of sessionStorage and routes, when created activity/flow has old { id } in url before we redirect to the new { id }

Reply:
let's add a comment for that

Reply:
done